src/applications/vaos/new-appointment/components/ReviewPage/CommunityCareSection.jsx

```diff
@@ -4,20 +4,24 @@ import ReasonForAppointmentSection from './ReasonForAppointmentSection';
 import PreferredDatesSection from './PreferredDatesSection';
 import PreferredProviderSection from './PreferredProviderSection';
 import SelectedProviderSection from './SelectedProviderSection';
+import { useSelector } from 'react-redux';
+import { selectFeatureCCIterations } from '../../../redux/selectors';
+
+export default function CommunityCareSection({ data, vaCityState }) {
+  const provider = data.communityCareProvider;
+  const hasProvider =
+    !!provider && !!Object.keys(data.communityCareProvider).length;
+  const myData = { ...data, hasCommunityCareProvider: hasProvider };
+  const featureCCIteration = useSelector(selectFeatureCCIterations);
 
-export default function CommunityCareSection({
-  data,
-  vaCityState,
-  hasResidentialAddress,
-}) {
   return (
     <>
       <PreferredDatesSection data={data} />
       <hr aria-hidden="true" className="vads-u-margin-y--2" />
-      {!hasResidentialAddress && (
-        <PreferredProviderSection data={data} vaCityState={vaCityState} />
+      {!featureCCIteration && (
+        <PreferredProviderSection data={myData} vaCityState={vaCityState} />
       )}
-      {hasResidentialAddress && (
+      {featureCCIteration && (
```

Comment:
How did you know to substitute `hasResidentialAddress` with `featureCCIteration`?

Reply:
Great question!. I had the same concern so I had a conversation with Lauren about this and I don't think the having an address condition was ever correct. So, I going to try this fix and hopefully I have not broken anything else!